scripts/doc-checker.py

```diff
@@ -0,0 +1,124 @@
+"""
+Example usage:
+python scripts/doc-checker.py asv_bench/benchmarks/utils.py modin/pandas
+
+Notes
+-----
+    * the script can take some pathes to files or to directories
+"""
+
+import click
+import subprocess
+
+
+def numpydoc_validate(path):
+    import os
+    import ast
+    from numpydoc.validate import validate
```

Comment:
why those things are not imported at module-level?

Reply:
fixed